src/server/api/controllers/batch.controller.js

```diff
@@ -0,0 +1,23 @@
+const knex = require('../../config/db');
+const Roles = require('../../constants');
+
+const Error = require('../lib/utils/http-error');
+
+const deleteSpecificBatchById = async (batchId, res, next) => {
+  try {
+    const user = (await Roles.SUPER_ADMIN) === 'super_admin';
+    if (!user) return next(new Error('User have no rights to delete'));
```

Comment:
this code is not required, as it is taken care by the middleware checkIfAuthorized and I see that it is added to the endpoint.
Also please check there are two different files for same endpoint in controllers folder and also routes folder.

Reply:
yes now it is resolved